pages/dashboard/dashboard-page.js

```diff
@@ -195,7 +195,7 @@ exports.DashboardPage = class DashboardPage extends BasePage {
     );
     this.whatNewsHeader = page.getByText('What’s new in Penpot?', { exact: true });
     this.pluginModalHeader = page.getByText(
-      'Build Plugins and enhance your workflow',
+      'Viewer role, designed to enhance collaboration',
```

Comment:
This force us to fix line this on the next release, and the name for `pluginModalHeader` no longer represents its content.

Reply:
This is a temporary solution to prevent the tests from failing for now

Reply:
Hi @chalapkoStanislav, how do you see the [proposed changes](https://github.com/penpot/penpotqa/commit/3be0255117c22de8821bcc0214afd6ebc1d744bd), which in summary are:
* Remove the `pluginModalHeader` dependency to work with whatever the text we find
* Add a loop to click the "continue" button the times required
* Make the names generic for future features pop-ups modals

Do you think it could work in case we forgot to disable the modals in PRE? If you agree, feel free to take any ideas from here, fix it, or evolve it to unblock the PR.